File: js/task1c/S1_SW_02.js
function runCommand(cmd) {
  switch (cmd) {
    case "start":
      return "Service started";
    case "stop":
      return "Service stopped";
    case "status":
      return "Service status unknown";
    default:
      return "Unknown command";
  }
}

const testCommands = ["start", "status", "restart"];

for (const cmd of testCommands) {
  console.log(`runCommand("${cmd}") =>`, runCommand(cmd));
}

File: python/task1c/S1_SW_02.py
def runCommand(cmd):
  if cmd == "start":
      return "Service started"
  if cmd == "stop":
      return "Service stopped"
  if cmd == "status":
      return "Service status unknown"
  return "Unknown command"

testCommands = ["start", "status", "restart"]

for cmd in testCommands:
  print(f'runCommand("{cmd}") =>', runCommand(cmd))
